lib/diazo/tests/test_wsgi.py

```diff
@@ -154,12 +154,12 @@ def _testfile(filename):
 class TestXSLTMiddleware(unittest.TestCase):
 
     def test_transform_filename(self):
-        import tempfile
-        import os
-
         from diazo.wsgi import XSLTMiddleware
         from webob import Request
 
+        import os
+        import tempfile
+
```

Comment:
Is this re-ordering working as intended?

In Zope we sort std-lib before other imports.

In Plone I thought it is completely alphabetically? (I remember Jens mentioned that.)